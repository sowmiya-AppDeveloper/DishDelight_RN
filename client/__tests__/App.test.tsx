// /**
//  * @format
//  */

// import 'react-native';
// import React from 'react';
// import App from '../src/Screens/App/App';

// // Note: import explicitly to use the types shiped with jest.
// import {it} from '@jest/globals';

// // Note: test renderer must be required after react-native.
// import renderer from 'react-test-renderer';

// it('renders correctly', () => {
//   renderer.create(<App />);
// });

// **
//  * @format
//  */

// import React from 'react';
// import 'react-native';

// // Note: import explicitly to use the types shiped with jest.
// import {it} from '@jest/globals';

// // Note: test renderer must be required after react-native.
// import {FlatList, Text} from 'react-native';
// import renderer from 'react-test-renderer';

// it('renders the Flatlist componenet', () => {
//   const home = renderer
//     .create(
//       <FlatList
//         data={['item!', 'item2', 'item3']}
//         keyExtractor={item => item}
//         renderItem={({item}) => <Text>{item}</Text>}
//       />,
//     )
//     .toJSON();

//   expect(home).toMatchSnapshot();
// });
