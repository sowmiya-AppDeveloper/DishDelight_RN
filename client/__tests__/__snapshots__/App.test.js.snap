// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Flatlist componenet 1`] = `
<RCTScrollView
  data={
    [
      "item!",
      "item2",
      "item3",
    ]
  }
  getItem={[Function]}
  getItemCount={[Function]}
  keyExtractor={[Function]}
  onContentSizeChange={[Function]}
  onLayout={[Function]}
  onMomentumScrollBegin={[Function]}
  onMomentumScrollEnd={[Function]}
  onScroll={[Function]}
  onScrollBeginDrag={[Function]}
  onScrollEndDrag={[Function]}
  removeClippedSubviews={false}
  renderItem={[Function]}
  scrollEventThrottle={50}
  stickyHeaderIndices={[]}
  viewabilityConfigCallbackPairs={[]}
>
  <View>
    <View
      onFocusCapture={[Function]}
      onLayout={[Function]}
      style={null}
    >
      <Text>
        item!
      </Text>
    </View>
    <View
      onFocusCapture={[Function]}
      onLayout={[Function]}
      style={null}
    >
      <Text>
        item2
      </Text>
    </View>
    <View
      onFocusCapture={[Function]}
      onLayout={[Function]}
      style={null}
    >
      <Text>
        item3
      </Text>
    </View>
  </View>
</RCTScrollView>
`;
